test/e2e/storage/testsuites/provisioning.go

```diff
@@ -225,6 +225,26 @@ func (p *provisioningTestSuite) defineTests(driver TestDriver, pattern testpatte
 		}
 		l.testCase.TestDynamicProvisioning()
 	})
+	ginkgo.It("should provision storage with pvc data source [Feature:VolumePVCDataSource]", func() {
+		if !dInfo.Capabilities[CapPVCDataSource] {
+			framework.Skipf("Driver %q does not support cloning - skipping", dInfo.Name)
+		}
+
+		init()
+		defer cleanup()
+
+		dc := l.config.Framework.DynamicClient
+		dataSource, cleanupFunc := preparePVCDataSourceForProvisioning(framework.NodeSelection{Name: l.config.ClientNodeName}, l.cs, dc, l.pvc, l.sc)
+		defer cleanupFunc()
```

Comment:
maybe call this dataSourceCleanup to be more clear

Reply:
changed to "dataSourceCleanup", thanks.